<?xml version='1.0' encoding='UTF-8'?>
<root version="2.3.8">
 <tabbed_widget name="Main Window" parent="main_window">
  <Tab tab_name="TimeDelayEstimator" containers="1">
   <Container>
    <DockSplitter count="2" sizes="0.5;0.5" orientation="-">
     <DockSplitter count="3" sizes="0.332117;0.335766;0.332117" orientation="|">
      <DockSplitter count="2" sizes="0.5;0.5" orientation="-">
       <DockArea name="...">
        <plot style="Lines" mode="TimeSeries">
         <range left="0.000000" top="1.000000" bottom="0.000000" right="1.000000"/>
         <limitY/>
        </plot>
       </DockArea>
       <DockArea name="...">
        <plot style="Lines" mode="TimeSeries">
         <range left="0.000000" top="1.000000" bottom="0.000000" right="1.000000"/>
         <limitY/>
        </plot>
       </DockArea>
      </DockSplitter>
      <DockSplitter count="2" sizes="0.5;0.5" orientation="-">
       <DockArea name="...">
        <plot style="Lines" mode="TimeSeries">
         <range left="0.000000" top="1.000000" bottom="0.000000" right="1.000000"/>
         <limitY/>
        </plot>
       </DockArea>
       <DockArea name="...">
        <plot style="Lines" mode="TimeSeries">
         <range left="0.000000" top="1.000000" bottom="0.000000" right="1.000000"/>
         <limitY/>
        </plot>
       </DockArea>
      </DockSplitter>
      <DockSplitter count="2" sizes="0.5;0.5" orientation="-">
       <DockArea name="...">
        <plot style="Lines" mode="TimeSeries">
         <range left="0.000000" top="1.000000" bottom="0.000000" right="1.000000"/>
         <limitY/>
        </plot>
       </DockArea>
       <DockArea name="...">
        <plot style="Lines" mode="TimeSeries">
         <range left="0.000000" top="1.000000" bottom="0.000000" right="1.000000"/>
         <limitY/>
        </plot>
       </DockArea>
      </DockSplitter>
     </DockSplitter>
     <DockSplitter count="3" sizes="0.332117;0.335766;0.332117" orientation="|">
      <DockSplitter count="2" sizes="0.5;0.5" orientation="-">
       <DockArea name="...">
        <plot style="Lines" mode="TimeSeries">
         <range left="0.000000" top="1.000000" bottom="0.000000" right="1.000000"/>
         <limitY/>
        </plot>
       </DockArea>
       <DockArea name="...">
        <plot style="Lines" mode="TimeSeries">
         <range left="0.000000" top="1.000000" bottom="0.000000" right="1.000000"/>
         <limitY/>
        </plot>
       </DockArea>
      </DockSplitter>
      <DockSplitter count="2" sizes="0.5;0.5" orientation="-">
       <DockArea name="...">
        <plot style="Lines" mode="TimeSeries">
         <range left="0.000000" top="1.000000" bottom="0.000000" right="1.000000"/>
         <limitY/>
        </plot>
       </DockArea>
       <DockArea name="...">
        <plot style="Lines" mode="TimeSeries">
         <range left="0.000000" top="1.000000" bottom="0.000000" right="1.000000"/>
         <limitY/>
        </plot>
       </DockArea>
      </DockSplitter>
      <DockSplitter count="2" sizes="0.5;0.5" orientation="-">
       <DockArea name="...">
        <plot style="Lines" mode="TimeSeries">
         <range left="0.000000" top="1.000000" bottom="0.000000" right="1.000000"/>
         <limitY/>
        </plot>
       </DockArea>
       <DockArea name="...">
        <plot style="Lines" mode="TimeSeries">
         <range left="0.000000" top="1.000000" bottom="0.000000" right="1.000000"/>
         <limitY/>
        </plot>
       </DockArea>
      </DockSplitter>
     </DockSplitter>
    </DockSplitter>
   </Container>
  </Tab>
  <currentTabIndex index="0"/>
 </tabbed_widget>
 <use_relative_time_offset enabled="1"/>
 <!-- - - - - - - - - - - - - - - -->
 <!-- - - - - - - - - - - - - - - -->
 <Plugins>
  <plugin ID="DataLoad CSV">
   <default time_axis=""/>
  </plugin>
  <plugin ID="DataLoad ROS2 bags">
   <use_header_stamp value="false"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="100"/>
  </plugin>
  <plugin ID="DataLoad ULog"/>
  <plugin ID="LSL Subscriber"/>
  <plugin ID="MQTT Subscriber"/>
  <plugin ID="ROS2 Topic Subscriber">
   <use_header_stamp value="false"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="true"/>
   <selected_topics>
    <topic name="/vehicle/status/accel_cmd_delay"/>
    <topic name="/vehicle/status/brake_cmd_delay"/>
    <topic name="/vehicle/status/steer_cmd_delay"/>
   </selected_topics>
  </plugin>
  <plugin ID="UDP Server"/>
  <plugin ID="WebSocket Server"/>
  <plugin ID="ZMQ Subscriber"/>
  <plugin status="idle" ID="CSV Exporter"/>
  <plugin status="idle" ID="ROS2 Topic Re-Publisher"/>
 </Plugins>
 <!-- - - - - - - - - - - - - - - -->
 <previouslyLoaded_Datafiles/>
 <!-- - - - - - - - - - - - - - - -->
 <customMathEquations/>
 <snippets/>
 <!-- - - - - - - - - - - - - - - -->
</root>
